src/jit/gentree.h

```diff
@@ -615,6 +615,12 @@ struct GenTree
         return reg;
     }
 
+    regNumber GetRegNum() const
+    {
+        assert((gtRegTag == GT_REGTAG_NONE) || !isContained());
```

Comment:
I am even more opposed to introducing new uses of `isContained()` into `GenTree`, as it is not a property that makes any sense prior to `Lowering`, and even then it doesn't *really* make sense until after register allocation.